src/App.tsx

```diff
@@ -46,7 +67,22 @@ const App: React.FC = () => {
       <AuthContext.Provider value={{ token, setToken }}>
         <WebsocketContext.Provider value={websocketClient}>
           <ThemeProvider theme={theme}>
+            <Global
+              styles={css({
+                body: {
+                  color: 'text',
+                  bg: 'background',
+                },
+              })}
+            />
             <Router />
+
+            <button
+              onClick={e => {
+                const next = mode === 'dark' ? 'light' : 'dark'
```

Comment:
Whether my suggestions are followed or not, I'd at the very least recommend we clean up the magic strings of `'dark'` and `'light'` throughout the file

Reply:
I should have mentioned this in the PR, but really, this bit was just to prove that it worked. This code, which proves that a button could be pushed to "toggle" a color mode, should live somewhere else in the app—like a settings view or something. Therefore, it is at that time, I will take your feedback into consideration! ;)